5calls/app/src/main/java/org/a5calls/android/a5calls/controller/MainActivity.java

```diff
@@ -213,14 +227,26 @@ public void onCallReported() {
         AppSingleton.getInstance(getApplicationContext()).getJsonController().registerStatusListener(mStatusListener);
     }
 
+    private void saveIssuesToDatabaseForUpgrade(List<Issue> issues) {
```

Comment:
The db update logic should probably live in the model layer, maybe the dbHelper.